Camomile/tools/iana.ml

```diff
@@ -62,7 +59,6 @@ let () =
                    Some (CharEncoding.name_of enc)
                  with Not_found ->
                  try
-                   let s = String.uppercase s in
```

Comment:
Why is it ok to remove this? (I'm not very familiar with this code)

Reply:
It wasn't used anywhere. After compiling this file, some warnings are revealed.

Reply:
Ok. I'm just thinking that the original author must have put this `String.uppercase` for a reason. 

Is the problematic warning the deprecation warning? If yes, I suggest to do the same as we do in `Stdune`:

```ocaml
module String = struct
  include struct
    [@@@warning "-32-3"]
    let uppercase_ascii   = String.uppercase
  end
  include String
end
```

and use `String.uppercase_ascii` here.

Reply:
I think the variable `s` here was just unused. So unless `uppercase` is called for its side effect, it can be removed.

Reply:
It looks used at the line just after. It seems that this code is doing:

- lookup `s`
- if not found, lookup `uppercase s`

Reply:
Ah yeah, I don't know what I was thinking. I reverted this change.